### PR TITLE
Add state constants for IdleStateHandler

### PR DESCRIPTION
**Pull request by JoeCqupt**
Motivation:

eliminate magic number

Modification:

Add state constants for IdleStateHandler

Result:

nothing

### CODE REVIEW

**Comment by JoeCqupt** [comment]
@normanmaurer  PTAL

File: handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java [hunk 1]
```diff
@@ -124,7 +124,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
     private Future<?> allIdleTimeout;
     private boolean firstAllIdleEvent = true;
 
-    private byte state; // 0 - none, 1 - initialized, 2 - destroyed
+    private byte state;
+    private static final byte  ST_INITIALIZED = 1;
```
Comment (hyperxpro) [inline handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java:128]:
```suggestion
    private static final byte ST_INITIALIZED = 1;
```

File: handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java [hunk 2]
```diff
@@ -124,7 +124,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
...
+    private static final byte  ST_DESTROYED = 2;
```
Comment (hyperxpro) [inline handler/src/main/java/io/netty/handler/timeout/IdleStateHandler.java:129]:
```suggestion
    private static final byte ST_DESTROYED = 2;
```

**Review by hyperxpro — CHANGES_REQUESTED** [review]

**Comment by JoeCqupt** [comment]
> @JoeCqupt did you sign our ICLA https://netty.io/s/icla ?

I have already signed it

**Comment by normanmaurer** [comment]
@JoeCqupt thanks a lot

**Pull request merged**